package.json

```diff
@@ -4,7 +4,8 @@
   "description": "The back end of the worriorz",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "coverage": "nyc report --reporter=text-lcov | coveralls"
   },
```

Comment:
Thanks @irfiacre ...

The **test** script is being included in the PR of @kabundege, so I didn't change it so that we can avoid conflicts. This is why I need to wait for @kabundege to finish and merge his PR before I can merge mine.

About Heroku, I believe Heroku runs the `npm start` script.

Reply:
@irfiacre I have just continued to make the changes as your feedback suggested (see **test** script in `package.json`) because it's getting late and I don't see the Travis CI pull request getting resolved.